add doc on how to exclude an SSC rule (#1850)
semgrep/semgrep-docs: docs/kb/semgrep-supply-chain/exclude-rule.md

---
tags:
  - Semgrep Supply Chain
description: Exclude a Semgrep Supply Chain rule from a scan
---

# How to exclude a Semgrep Supply Chain rule from a scan

To troubleshoot a problematic rule or to remove a rule that's too noisy, you can exclude a specific rule from being run during a Semgrep Supply Chain scan using the `--exclude-rule` flag:

```console
semgrep ci --exclude-rule <rule_ID>
```

> The `--exclude-rule` flag requires the rule ID as a parameter. You can get this value as follows:

Reviewer: ```suggestion
The `--exclude-rule` flag requires the rule ID as a parameter. To retrieve this value:
```
Or

1. Sign in to [<i class="fas fa-external-link fa-xs"></i> Semgrep AppSec Platform](https://semgrep.dev/login), and go to **Supply Chain**.

> 2. Select the finding whose details you want to view details:

Reviewer: ```suggestion
2. Select the finding whose details you want to view:
```
